controllers/model/model_controller.go

```diff
@@ -33,6 +33,7 @@ import (
 
 	modelv1 "github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/model"
 	. "github.com/aws/amazon-sagemaker-operator-for-k8s/controllers"
+	"github.com/aws/amazon-sagemaker-operator-for-k8s/controllers/controllertest"
```

Comment:
Weird that we are using a test package in a non-test location? Can we move the `ToStringPtr` method to a common package? Maybe `/controllers`?

Reply:
ya that is weird   
it is being used in a lot of places  
i can add this task to our backlog